ImageFeed/ImagesList/ImagesListViewController.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  ViewController.swift
+//  ImageFeed
+//
+//  Created by Алексей Гвоздков on 30.04.2023.
+//
+
+import UIKit
+
+class ImagesListViewController: UIViewController {
+    
+    @IBOutlet private var tableView: UITableView!
+    
+    private let photosNames: [String] = Array(0..<20).map{ "\($0)" }
```

Comment:
Нужны пробелы вокруг открывающей фигурной скобки:
```suggestion
    private let photosNames: [String] = Array(0..<20).map { "\($0)" }
```